src/internal/Common.h

```diff
@@ -25,6 +25,8 @@ typedef Range<const char*> StringPiece;
 
 namespace rsocket {
 
+constexpr int64_t kMaxRequestN = std::numeric_limits<int32_t>::max();
```

Comment:
https://github.com/rsocket/rsocket/blob/master/Protocol.md#request_n-frame-0x08
If we check here, the limit is 2^31-1. Which is not the same as the limit of yarpl. I have created another issue to solve this
https://github.com/rsocket/rsocket-cpp/issues/561